### PR TITLE
Move Medallia feedback button to bottom of main content

### PR DESCRIPTION
**Pull request by aubreyarcangel**
## Description
Addresses [#27934](https://github.com/department-of-veterans-affairs/va.gov-team/issues/27934)

This PR adds a new Medallia feedback button to the bottom of the main content. 

Steps taken:

1. Created Medallia feedback button `src/site/includes/medallia-feedback-button.drupal.liquid` 
2. Combined the `last-updated` component with the the new `Medallia feedback button` in `"src/site/includes/above-footer-elements.drupal.liquid"`
3. Added new styles to match [mock up designs](https://www.sketch.com/s/65948b03-95a4-4bf7-b8fa-75caff6bfa4a/a/52LxYvE) and work with pages that have the `<va-back-to-top>`  web component

Templates where Medallia Feedback button is added | Url that uses the template
-- | --
src/site/layouts/faq_multiple_q_a.drupal.liquid | /resources/signing-in-to-vagov/
src/site/layouts/landing_page.drupal.liquid | /family-member-benefits
src/site/layouts/page.drupal.liquid | /health-care/covid-19-vaccine-tag/
src/site/layouts/support_resources_detail_page.drupal.liquid | /resources/request-a-discharge-upgrade-or-correction/
src/site/layouts/vet_center.drupal.liquid | /north-orange-county-vet-center/
src/site/layouts/campaign_landing_page.drupal.liquid | /initiatives/recognizing-lgbtq-veterans-during-pride-month/
src/site/layouts/step_by_step.drupal.liquid | /resources/how-to-check-your-va-claim-appeal-or-decision-review-status-online
src/site/layouts/basic_landing_page.drupal.liquid | /resources/
src/site/layouts/health_care_local_facility.drupal.liquid | /pittsburgh-health-care/locations/pittsburgh-va-medical-center-university-drive/
src/site/layouts/va_form.drupal.liquid | /find-forms/about-form-21-10210/
src/site/layouts/support_resources_article_listing.drupal.liquid | /resources/va-account-and-profile/
src/site/layouts/health_care_region_page.drupal.liquid | /providence-health-care/
src/site/layouts/health_care_region_detail_page.drupal.liquid | /portland-health-care/programs/whole-health/active-movement-classes/
src/site/layouts/event.drupal.liquid | /cincinnati-health-care/events/wellness-strength-for-life-2/
src/site/layouts/vamc_operating_status_and_alerts.drupal.liquid | /sioux-falls-health-care/operating-status
src/site/layouts/vamc_system_policies_page.drupal.liquid | /bronx-health-care/policies/
src/site/layouts/event_listing.drupal.liquid | /pittsburgh-health-care/events/
src/site/layouts/page-react.html | /pension/application/527EZ/

## Testing done

- Building & serving locally.

## Screenshots
/resources/signing-in-to-vagov/
Screen Shot 2022-02-06 at 8 58 04 PM

/find-locations/
Screen Shot 2022-02-06 at 8 59 33 PM

/health-care/
Screen Shot 2022-02-06 at 9 00 32 PM

 /pittsburgh-health-care/
Screen Shot 2022-02-06 at 9 02 41 PM

## Acceptance criteria
- [x] Button matches designs (which includes co-locating button with last updated and back to top button on pages they are displayed)
- [x] Button launches correct survey modal  depending on URL
- [x] All launch-blocking 508 issues are addressed

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by ndsprinkle — APPROVED** [review]

**Comment by jchilders** [comment]
When does this need to be reviewed by?

**Comment by aubreyarcangel** [comment]
> When does this need to be reviewed by?

As soon as you can. We're hoping to get it in staging to start testing.

File: src/site/layouts/page-breadcrumbs.html [hunk 1]
```diff
@@ -27,3 +27,4 @@
 
 </div>
 {% include "src/site/includes/footer.html" %}
+{% include "src/site/includes/debug.drupal.liquid" %}
```
Comment (srsuddath) [inline src/site/layouts/page-breadcrumbs.html:30]: do we need this here? I am unfamiliar with that the debug template is doing?

Comment (aubreyarcangel) [reply src/site/layouts/page-breadcrumbs.html:30]: No. I'll remove it.

**Review by GcioNapoleon — APPROVED** [review]

File: src/applications/registry.json [hunk 2]
```diff
@@ -1253,4 +1253,4 @@
       "layout": "page-react.html"
     }
   }
-]
+]
```
Comment (RudyOnRails) [inline src/applications/registry.json:1256]: Should we keep the new line at the end of the file?

**Review by RudyOnRails — APPROVED** [review]
LGTM

**Review by mdewey — APPROVED** [review]
LGTM!

**Pull request merged**